### ISSUE TITLE
Table is resizing columns and removing content when pressing "add left/right column"

### ISSUE THREAD

**Issue by LeanidBusel** [TypeCellOS/BlockNote#864]
**To Reproduce**
If write something in a cell and then add a new column. Content will be removed.
If change the width of an empty column and then add a new column. Width will be returned back

[table_test.webm](https://github.com/TypeCellOS/BlockNote/assets/173772763/5a6ff5a0-875f-42cf-9459-d9a5f6388e47)

**Misc**
- BlockNote: 0.14.1

**Comment by thedevangvishnu** [comment]
Yes, this issue is weird. Firstly, there is this strange resizing/shrinking in the size of the cell and secondly, adding of new columns is messing it up even more. Sometimes, the new column does get added (without deleting the existing cell data) and most of the times, adding new column completely deletes the previous data. The behavior is very inconsistent.

**Comment by matthewlipski** [comment]
Thanks for reporting! This seems like something that should be addressed in #856

**Comment by LeanidBusel** [comment]
any updates? Also when adding a new row is deletes the previous data. It can be reproduce in Basic Setup example.

**Comment by nperez0111** [comment]
I will address this with https://github.com/TypeCellOS/BlockNote/pull/1429

**Comment by nperez0111** [comment]
Resolved with https://github.com/TypeCellOS/BlockNote/pull/1429
